src/VersionManager.cs

```diff
@@ -19,68 +20,162 @@ internal class VersionManager
         /// <summary>
         /// The latest version of DarkRift.
         /// </summary>
-        private static string latestDarkRiftVersion;
+        private static Version latestDarkRiftVersion;
 
         /// <summary>
         /// Gets the path to a specified installation, downloading it if required.
         /// </summary>
         /// <param name="version">The version number required.</param>
         /// <param name="pro">Whether the pro version should be used.</param>
-        /// <param name="netStandard">Whether the .NET Standard build should be used.</param>
+        /// <param name="platform">Whether the .NET Standard build should be used.</param>
         /// <returns>The path to the installation, or null, if it cannot be provided.</returns>
-        public static string GetInstallationPath(string version, ServerTier tier, ServerPlatform platform)
+        public static string GetInstallationPath(Version version, ServerTier tier, ServerPlatform platform)
```

Comment:
Changing the behaviour of this method so it doesn't automatically download versions will break a number of things such as the `run` command as they will no longer pull the version they need (and most will probably just `NullPointerException` out now to be honest)